test/bin/scenario_container.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
+    echo "This script must be sourced, not executed."
+    exit 1
+fi
+
+process_kickstart_into_script() {
+    local -r container_name="$1"
+    local -r file=${2:-kickstart.ks}
+
+    local -r output_dir="${SCENARIO_INFO_DIR}/${SCENARIO}/vms/${container_name}"
+
+    while IFS= read -r line; do
+        if [[ $line =~ ^%include[[:space:]]+(.+) ]]; then
+            process_kickstart_into_script "${container_name}" "${BASH_REMATCH[1]}"
+        else
+            echo "$line" >> "${output_dir}/init.sh"
+        fi
+    done < "${output_dir}/${file}"
+}
+
+create_container_entrypoint() {
+    local -r name="$1"
+    local -r output_dir="${SCENARIO_INFO_DIR}/${SCENARIO}/vms/${name}/"
+
+    cat << EOF > "${output_dir}/init.sh"
+#!/bin/bash
+set -xeuo pipefail
+
+cat /proc/sys/kernel/random/uuid > /proc/sys/kernel/random/boot_id
+
+if [ -e /var/container-init-ok ]; then
```

Comment:
Should it be in /var/run so that file is not persisted after reboot?

Reply:
We don't want these on each reboot. Only the first one. These kickstart commands are written to be executed once during installation, they don't handle rerunning well.

Reply:
I see. Can you store the file under a directory, e.g. `/var/adm` or like?